chapters/http-requests.adoc

```diff
@@ -493,9 +476,6 @@ provides the following benefits:
 ** No external documents or grammars needed
```

Comment:
Are we going to still use 'Zalando' word in examples? If no, then we need to replace 'Zalando' with something in line 435: * `name=Zalando`, to query for elements based on property equality
_(and maybe in line 52)_

Reply:
Thanks Khrystyna. I couldn't find these places in the code, but I remember specifically changing that query param example to name=IST on Friday, so should be fixed now